.github/workflows/dashwallet.yml

```diff
@@ -68,13 +68,19 @@ jobs:
 
     - name: Decrypt secrets
       run: |
-        gpg -d --passphrase "${{ secrets.GOOGLE_SERVICES_KEY }}" --batch .deploy/google-services.json.gpg > wallet/google-services.json
         gpg -d --passphrase "${{ secrets.SERVICE_PROPERTIES_KEY }}" --batch .deploy/service.properties.gpg > service.properties
         gpg -d --passphrase "${{ secrets.APP_DISTRIBUTION_KEY }}" --batch .deploy/app-distribution-key.json.gpg > .deploy/app-distribution-key.json
         gpg -d --passphrase "${{ secrets.KEYSTORE_KEY }}" --batch .deploy/keystore.jks.gpg > .deploy/keystore.jks
         gpg -d --passphrase "${{ secrets.LOCAL_PROPERTIES_KEY }}" --batch .deploy/local.properties.gpg > local.properties
         gpg -d --passphrase "${{ secrets.GC_STORAGE_SERVICE_ACCOUNT_KEY }}" --batch .deploy/gc-storage-service-account.json.gpg > .deploy/gc-storage-service-account.json
 
+    - name: Extract Secrets
+      run: |
+        echo "$GOOGLE_SERVICES_JSON" > wallet/google-services.json
+        cat wallet/google-services.json
```

Comment:
Won't this line put the contents of `google-services.json` into the build log for everyone to see?

Reply:
As it turns out, it outputs this.  Nevertheless, this `cat` command is not necessary.
```
***
  ***
    ***
    ***
    ***
    ***
  ***
  ```